kod/object/item/passitem/specwand.kod

```diff
@@ -36,6 +43,14 @@ classvars:
    viObject_flags = APPLY_YES | GETTABLE_YES
    viItem_type = ITEMTYPE_WAND
 
+   vbShow_condition = TRUE
+   vrCondition_exc =   wand_condition_exc
```

Comment:
These 6 classvars need to be updated to fix the issue I called out in my comments. They're referencing the incorrect resources.